datalad/support/json_py.py

```diff
@@ -20,13 +20,7 @@
 from os import makedirs
 import os
 import os.path as op
-
-# wrapped below
-from simplejson import load as jsonload
-from simplejson import dump as jsondump
-# simply mirrored for now
-from simplejson import loads as json_loads
-from simplejson import JSONDecodeError
+import json
```

Comment:
To minimize impact and possibly avoid some part of this PR diff I would have:

```suggestion
import json
from json import JSONDecodeError
```

Reply:
This would not be sufficient. There would need to be replacements for `jsonload`, `jsondump`, `json_loads` too.

I personally think a clean cut is better than yet another intermediate import target to preserve forever.